doc/hoomd-examples.rst

```diff
@@ -178,3 +178,33 @@ Logged data must be a convertible to a numpy array of a supported type:
         s.particles.N = 4
         s.log['invalid'] = dict(a=1, b=5)
         t.append(s)
+
+Use multiprocessing with HOOMDTrajectory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. ipython:: python
+
+   import multiprocessing as mp
+
+   def cnt_part(args):
+      t, slc = args
+      Np = []
+      for traj_frame in t[slc]:
+         Np.append(len(traj_frame.particles.position))
+      return Np
+
+   with gsd.hoomd.open(name='test.gsd', mode='rb') as t:
+      frames = len(t)
+      cores = mp.cpu_count()
+      frames_per_core = frames // cores
```

Comment:
Is this the typical and recommended way to use multiprocessing? I would have thought it would be better to pass one (or a small number) of frames at a time to allow for dynamic load balancing. Are you trying to avoid the file open overhead time in the inner loop? In any case, a tutorial level example should be as simple as possible.

Reply:
I would say it depends on the load and the overhead for file opening and process creating. That was the original purpose for large trajectories. I can make it single frame easily enough though, and it would be clearer.